babycenter/templates/core/section_page.html

```diff
@@ -7,7 +7,7 @@
   <h1 class="l-indent heading heading{{self.get_effective_extra_style_hints}}">
     {{self.title}}
   </h1>
-  {% load_child_articles_for_section self as articles %}
+  {% load_child_articles_for_section self count=10 as articles %}
```

Comment:
Lets make this `13` and check with Botswang - I seem to recall they wanted it 13+